Add subscription_capacity table

kahowell opened the pull request:
The intention for a given subscription, which can be identified by a
subscription ID, to collect each applicable product's contributing
socket count, and whether the subscription provides unlimited guests
(VDC-style SKUs).

This does not handle any local caching of product data, which we may
elect to do in the future in order to improve performance.

This table is intentionally denormalized, as we intend to start out by
using denormalized views of subscription data from Candlepin pools.

mstead commented on src/main/resources/liquibase/201909162045-add-capacity-table.xml:
```diff
@@ -0,0 +1,31 @@
+            tableName="capacity_account"
+            columnNames="account_number,product_id,subscription_id"/>
+    </changeSet>
+
```
Might want to consider adding some indexes for searching purposes - especially around the dates.

kahowell commented:
@mstead, addressed feedback...

d85ad5b to 957e608:

```diff
diff --git a/src/main/resources/liquibase/201909162045-add-capacity-table.xml b/src/main/resources/liquibase/201909162045-add-capacity-table.xml
index 60be4de..436ca4d 100644
--- a/src/main/resources/liquibase/201909162045-add-capacity-table.xml
+++ b/src/main/resources/liquibase/201909162045-add-capacity-table.xml
@@ -8,7 +8,7 @@
 
     <changeSet id="201909162045-1" author="khowell">
         <comment>Add table for account subscription capacity</comment>
-        <createTable tableName="capacity_account">
+        <createTable tableName="subscription_capacity">
             <column name="account_number" type="VARCHAR(32)"/>
             <column name="product_id" type="VARCHAR(32)"/>
             <column name="subscription_id" type="VARCHAR(255)"/>
@@ -22,10 +22,21 @@
     </changeSet>
 
     <changeSet id="201906181633-2" author="khowell">
-        <addPrimaryKey constraintName="cap_account_pkey"
-            tableName="capacity_account"
+        <addPrimaryKey constraintName="subs_cap_pkey"
+            tableName="subscription_capacity"
             columnNames="account_number,product_id,subscription_id"/>
     </changeSet>
 
+    <changeSet id="201906181633-3" author="khowell">
+        <createIndex indexName="subs_cap_begin_date_idx" tableName="subscription_capacity"
+            unique="false">
+            <column name="begin_date"/>
+        </createIndex>
+        <createIndex indexName="subs_cap_end_date_idx" tableName="subscription_capacity"
+            unique="false">
+            <column name="end_date"/>
+        </createIndex>
+    </changeSet>
+
 </databaseChangeLog>
 <!-- vim: set expandtab sts=4 sw=4 ai: -->
```
I decided to force push to avoid need to squash. :-)
